src/client/components/builder/QuestionsTab.tsx

```diff
@@ -42,6 +44,14 @@ const generateDefaultRadioField = (id: number): checker.Field => ({
   options: [{ label: 'Option 1', value: 0 }],
 })
 
+const generateDefaultDropdownField = (id: number): checker.Field => ({
+  id: `DL${id}`,
```

Comment:
Currently when you duplicate, the ID will be `DX` instead of `DLX`. This is an issue because of [our hacky way](https://github.com/opengovsg/checkfirst/blob/122c8e4b97ca17826d7f5d3cfa0337ebf9f71694/src/client/components/builder/BuilderField.tsx#L145) of doing duplicates which assume that the prefix for a field type is just 1 character long.

Reply:
Fixed that here:
https://github.com/opengovsg/checkfirst/pull/276/files#diff-15a8ab6b16cf6cdcbfc83b28f231671796c54b4641f02b0d4bd96be8f53801a4R84